lib/flight_scheduler/batch_script_runner.rb

```diff
@@ -114,7 +97,7 @@ def run
     # Kills the associated subprocess
     def cancel
       return unless @child_pid
-      Process.kill('SIGTERM', @child_pid)
+      Process.kill('SIGTERM', -@child_pid)
```

Comment:
FYI This line is incorrect. It is actually the `signal` that needs to be negative and not the PID. I think I had it like this in my other branch at one point in time, however I have subsequently changed it.

Don't fixing it here, I have corrected it in #25